github/github.go

```diff
@@ -154,6 +156,20 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+// RawType represents type of raw format of a request instead of JSON.
+type RawType int
```

Comment:
Maybe use `uint8` instead of `int` here for consistency with https://github.com/google/go-github/blob/466070b0580728e63bd1a415e0019639e55d7148/github/github.go#L684.